spinalcordtoolbox/centerline/optic.py

```diff
@@ -63,6 +64,17 @@ def centerline2roi(fname_image, folder_output='./', verbose=0):
     return fname_output
 
 
+def apply_intensity_normalization(fname_in, fname_out):
+    img = Image(fname_in)
+    img_normalized = img.copy()
+    p2, p98 = np.percentile(img.data, (2, 98))
```

Comment:
Isn't int16 range -32,768 to 32,767?

Reply:
I am sorry, I am not sure to understand: does the below comment helps? Otherwise, May I ask you to re-explain to me? Thank you

Reply:
```
Wouldn't it be better to fix the issue inside sct_image instead?
```
Maybe! But I am not sure to see how, Could you please help me? Merci

Reply:
sorry @charleygros i meant line 71 (below), where it says:  out_range=(0, 255) (equivalent 8bit unsigned), whereas later in the code the data is converted to int16, hence we loose precision. Or am i missing something?

Reply:
maybe [skimage.img_as_uint](http://scikit-image.org/docs/dev/api/skimage.html#skimage.img_as_uint) could be useful here?

Reply:
It seems to be a better solution, merci!

Just one question: The documentation says:
```
Negative input values will be clipped.
```
--> Could we assume that images will always have positive values?

Reply:
i suggest we do, but we should check for it. If there are neg values, a "warning" should be sent.